### PR TITLE
Chapter 7 

### PR DESCRIPTION
**Pull request by tenyiting**

### CODE REVIEW

**Comment by tenyiting** [comment]
ready

File: app/assets/javascripts/users.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
```
Comment (ngocvt-0484) [inline app/assets/javascripts/users.coffee]: remove it because not use

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/users.scss]: same, remove please

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
-  resources :posts
-  get 'static_pages/home'
-  get 'static_pages/help'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'users/show'
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove because not use

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,13 @@
...
+  get 'users/new'
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove because not use

File: test/controllers/users_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test "should get show" do
+    get users_show_url
+    assert_response :success
+  end
+
```
Comment (ngocvt-0484) [inline test/controllers/users_controller_test.rb]: dont need do test
so please remove this file

File: config/routes.rb (hunk 3 above)
Comment (tenyiting) [reply config/routes.rb]: Screenshot from 2021-09-10 12-56-09

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**